Kmeans_python/fit.py

```diff
@@ -1,7 +1,83 @@
-def fit(X_train, k):
+import pandas as pd
+import numpy as np
+
+def compute_distance(samples, centers):
+    
+    """
+    This computes the euclidean distance of each sample from all the cluster centers
+    
+    Parameters
+    ----------
+    
+    samples : numpy.ndarray
+    all the data points in our sample
+    
+    centers : numpy.ndarray
+    the centroids of the clusters already selected
+    
+    Returns
+    ---------
+    
+    numpy.ndarray
+    an array with all the samples and their distances from each of the cluster centers
+    
     """
-    This functions classifies the non-labeled data into a given number of clusters k.
-    It returns labels for each data point according to the cluster it belong and also 
+
+    k, num_features = centers.shape
+    n_samples = samples.shape[0]
+
+    distance = np.empty((n_samples, k))
+
+    for i in range(k):
+        
+        # taking the euclidean distance
+        distance[:, i] = np.linalg.norm(samples - centers[i, :], axis=1)
+    
+    return distance
+
+def random_init(samples, k, n_features):
+    
+    """
+    This initialize the cluster centers based on random assignment
+    
+    Parameters
+    ----------
+    
+    samples : numpy.ndarray
+    all the data points in our sample
+    
+    k : int
+    the number of clusters to be made
+    
+    n_features : int
+    the number of features in the sample
+    
+    Returns
+    ---------
+    
+    numpy.ndarray
+    an array with all the cluster centers and their location based on values of each feature
+    
+    """
+    
+    centers = np.empty((k, n_features))
+
+    max_features = np.max(samples, axis = 0)
+
+    min_features = np.min(samples, axis = 0)
+    
+    for i in range(k):
+            
+        centers[i,:] = min_features + np.random.random(n_features)*(max_features - min_features)/3 + \
+                                            np.random.random(n_features)*(max_features - min_features)/3 +  \
+                                                np.random.random(n_features)*(max_features - min_features)/3
+
+    return centers
+
+def fit(X_train, k, n_init = 10, max_iter = 200):
+    """
+    This function classifies the non-labeled data into a given number of clusters k using simple KMeans algorithm.
+    It returns labels for each data point according to the cluster it belongs and also 
     cluster centers. This is a type of unsupervised learning method to classify data.
```

Comment:
```suggestion
    """
    This function classifies the non-labeled data into a given number of clusters k using a simple KMeans algorithm.
    It returns labels for each data point according to the cluster it belongs to and also cluster centers. This is a type of unsupervised learning method to cluster data.
```

Reply:
Is the change only in the <return> character in the second sentence? If yes, I don't think it needs to be changed